lib/fastlane/actions/lcov.rb

```diff
@@ -28,13 +28,16 @@ def self.available_options
                                        env_name: "FL_LCOV_SCHEME",
                                        description: "Scheme of the project"),
 
+          FastlaneCore::ConfigItem.new(key: :arch,
+                                       env_name: "FL_LCOV_ARCH",
+                                       description: "The build arch where will search .gcda files"),
```

Comment:
@KrauseFx @xfreebird Why should have a default_value if is not a optional parameter?